nav2_planner/src/planner_server.cpp

```diff
@@ -27,6 +27,7 @@
 #include "builtin_interfaces/msg/duration.hpp"
```

Comment:
Check the CI linters https://app.circleci.com/pipelines/github/ros-planning/navigation2/6576/workflows/0be4de15-e62c-4770-bb9f-f931edc6fd6d/jobs/24252. You can reproduce locally with `ament_cpplint` and `ament_uncrustify`

Reply:
Done, however I am still getting errors about missing test results for the nav2 system tests. Not really sure how to handle those.

Reply:
@jwallace42 you can ignore the system tests backup/spin tests, those are just flaky, but the rest should work. Looks like at least 1 linting problem. 

You should also rebase / pull in main. It'll speed up CI and make the diff easier to process

Reply:
Done